src/SConscript

```diff
@@ -39,6 +39,7 @@ def scons():
                     'include/gurt/%s' % header)
 
     env.Append(CCFLAGS=['-DCART_VERSION=\\"' + CART_VERSION + '\\"'])
+    env.AppendUnique(CPPPATH=[Dir('cart/src/utest').srcnode()])
```

Comment:
I think it will add both source and build automatically if you just use Dir.  I also think this belongs down in cart/src/utest/Sconscript as a Dir('.')

Note that scons will read all of the SConscript's before building anything so even if this line is after the call that builds the gurt tests, it will affect the environment used to build those tests

Reply:
This will be fixed when moving cart.